model_templates/inference/python3_unstructured/custom.py

```diff
@@ -12,6 +12,7 @@ def score_unstructured(model, data, query, **kwargs):
     if isinstance(data, bytes):
         data = data.decode("utf8")
 
+    data = data.strip().replace("  ", " ")
     words_count = data.count(" ") + 1
```

Comment:
nit. could just be `len(data.split())` Default behavior splits on all whitespace